src/screens/Dashboard/Subscribe/style.scss

```diff
@@ -0,0 +1,58 @@
+@import "../../../scss/variables";
+@import "../../../scss/_base";
+
+#subscribe {
+  position: relative;
+  margin: 2rem 0;
+  height: 600px;
+
+  .content { 
+    position: absolute;
+    z-index: 1;
+    top: 12rem;
+    left: 9rem;
+    display: flex;
+    flex-direction: column;
+
+    .title {
+      font-size: 2.5rem;
+      font-family: $font-primary;
+    }
+
+    .inputContainer {
```

Comment:
We should switch up using camel case for CSS to using kebab case. Example className:  subscribe-content-title. Something similar to BEM, with maybe less underscores since those can get rather long.

Reply:
update to kebab case